cloud_build_prod.yaml

```diff
@@ -0,0 +1,21 @@
+# Provide instructions for google Cloud Build to auto-build flutter
+# dashboard to flutter-dashboard project. Auto-build will be triggered
+# by new commits to `master` branch.
+
+steps:
+
+  # Build app_flutter.
+  - name: gcr.io/flutter-dashboard/flutter
```

Comment:
How to update this docker image?

Reply:
I am considering adding a trigger to update that later. Now it was built from my local. Here is the tracking issue: https://github.com/flutter/flutter/issues/61043

Reply:
Is there a Dockerfile? It's better to check in the Dockerfile with this PR so that we know what's contained in the image.

Reply:
This is the repo: https://github.com/GoogleCloudPlatform/cloud-builders-community/tree/master/flutter
This info will be documented in the team doc.